deps.edn

```diff
@@ -1,9 +1,17 @@
 {:paths     ["src" "target/classes"]
  :deps
- {org.clojure/clojure           #:mvn{:version "1.8.0"}
+ {org.clojure/clojure           #:mvn{:version "1.10.3"}
+  org.clojure/core.cache        #:mvn{:version "1.0.207"}
   clj-stacktrace/clj-stacktrace #:mvn{:version "0.2.8"}
   clj-http/clj-http             #:mvn{:version "2.0.0"}
   cheshire/cheshire             #:mvn{:version "5.5.0"}
   environ/environ               #:mvn{:version "1.0.2"}
   org.clojure/data.json         #:mvn{:version "0.2.6"}}
```

Comment:
data.json received a good few enhancements lately, could you check if it can be bumped please?

Also, cheshire could be dropped if not needed. The rest of the deps could also be bumped if it doesn't cause problems.